frb_rust/src/rust2dart.rs

```diff
@@ -139,3 +145,120 @@ impl<T: IntoDart> StreamSink<T> {
         self.rust2dart().close_stream()
     }
 }
+
+/// Basically the Into trait.
+/// We need this separate trait because we need to implement int for Vec<T> etc.
```

Comment:
nit: would be great to explain a bit more why, or at least point to this PR

Reply:
i will expand the comment and link the pr